scripts/rollup/build-ghaction.js

```diff
@@ -0,0 +1,869 @@
+'use strict';
```

Comment:
Review note: diffed this against rollup/build.js locally. Its mostly a copy with some changes to how we get node count from the runner. LGTM, assume we'll remove rollup/build.js once this is stable.

Reply:
yeah I'm gonna remove the circleci hardcoding later in the stack and unfork these files